.github/workflows/backport-action.yml

```diff
@@ -0,0 +1,47 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+---
+name: Automatic backport action
+on:  # yamllint disable-line rule:truthy
+  pull_request_target:
+    types: ['labeled', 'closed']
+    branches: ['main']
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  backport:
+    name: Backport PR ${{ github.event.pull_request.number }}
+    if: >
+      github.event.pull_request.merged == true &&
+      contains(join(github.event.pull_request.labels.*.name, ' '), 'backport-to-v')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Execute Backport Action
+        uses: sorenlouv/backport-github-action@ad888e978060bc1b2798690dd9d03c4036560947
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          auto_backport_label_prefix: backport-to-
+
+      - name: Info log
```

Comment:
do we need these info?

Reply:
and debug step aswell?

Reply:
Yeah for quite a while (and maybe forever) we want to be able to look at the logs easily.

Reply:
okay fine lets keep it :)